src/reference/asciidoc/amqp.adoc

```diff
@@ -2821,6 +2821,57 @@ The third creates names like `custom.gen-MRBv9sqISkuCiPfOYfpo4g`.
 
 Of course, you can provide your own naming strategy bean.
 
+[[delayed-message-exchange]]
+==== Delayed Message Exchange
+
+_Version 1.6_ introduces support for the
+https://github.com/rabbitmq/rabbitmq-delayed-message-exchange[Delayed Message Exchange Plugin].
```

Comment:
Maybe we should mention the blog as https://www.rabbitmq.com/blog/2015/04/16/scheduling-messages-with-rabbitmq/ ?